docs/sites.yml

```
@@ -10055,3 +10055,17 @@
   built_by: Built by Rebels Ltd.
   built_by_url: https://builtbyrebels.com/
   featured: false
+- title: Kosmos & Kaos
+  main_url: https://www.kosmosogkaos.is/
+  url: https://www.kosmosogkaos.is/
+  description: >
+    A carefully designed user experience is damn good business.
```

```suggestion
    A carefully designed user experience is good business.
```

Another nit pick, this may not be as strong of a word in many countries but we want to make sure the content on our site is suitable for all young ages and can't assume they'd be exposed to some words. 

Maybe the description could be more general like "The homepage for Kosmos & Kaos..."?

---

Thanks @gillkyle  i've updated the description, and I'll let you know as soon as we've updated the homepage.

---

Thanks @arnorhs! Feel free to ping me whenever you have an update!